doc/source/dev/reviewer_guidelines.rst

```diff
@@ -7,19 +7,48 @@ Reviewer Guidelines
 Reviewing open pull requests (PRs) is very welcome and a valuable way to help
 increase the speed at which the project moves forward. We encourage everyone to
 get involved in the review process; it's also a great way to get familiar with
-the codebase. If we do not know you yet, consider introducing yourself before
-you start reviewing code.
+the codebase.
+
+Who can be a reviewer?
+======================
+
+You don't have to be a project member to contribute reviews. Consider being a
+reviewer if 
+
+- You have contributed to the NumPy project before, and feel like you can
+  comment on technical or organizational aspects of the PR; 
+- You haven't contributed before, but are an experienced NumPy user, and see a
+  pull request that touches a funcionality that you use often;
+- You can contribute comments about content, style, organization or good
+  documentation practices.  
+
+If we do not know you yet, consider introducing yourself before you start
+reviewing pull requests.
+
+Communication Guidelines
+========================
+
+- Every PR, good or bad, is an act of generosity. Opening with a positive
+  comment will help the author feel rewarded, and your subsequent remarks may be
+  heard more clearly. You may feel good also.
+- Begin if possible with the large issues, so the author knows they've been
+  understood. Resist the temptation to immediately go line by line, or to open
+  with small pervasive issues.
+- You are the face of the project, and NumPy some time ago decided `the kind of
+  project it will be <https://numpy.org/code-of-conduct/>`_: open, empathetic,
+  welcoming, friendly and patient. Be `kind
+  <https://youtu.be/tzFWz5fiVKU?t=49m30s>`_ to contributors.
+- Do not let perfect be the enemy of the good, particularly for documentation.
+  If you find yourself making many small suggestions, either open a PR against
+  the original branch, push changes to the contributor branch, or merge the PR
+  and then open a new PR against upstream.
+- If you need help writing replies in reviews, check out some `Standard replies
+  for reviewing
+  <https://scikit-learn.org/stable/developers/tips.html#saved-replies>`_.
 
 Reviewer Checklist
 ==================
 
-- Be patient and `kind <https://youtu.be/tzFWz5fiVKU?t=49m30s>`_ with
-  contributors.
-- When merging contributions, a committer is responsible for ensuring that those
-  meet the requirements outlined in the :ref:`Development process guidelines
-  <guidelines>` for NumPy. Also, check that new features and backwards
-  compatibility breaks were discussed on the `numpy-discussion mailing list
-  <https://mail.python.org/mailman/listinfo/numpy-discussion>`_.
 - Is the intended behavior clear under all conditions (e.g. unexpected inputs
   like empty arrays or nan/inf values)?
 - Is the code easy to read and low on redundancy? Should variable names be
```

Comment:
We don't have to tell reviewers to complain if they can't make sense of the code. But we want them to make constructive suggestions, not just complain, and the list of failures and improvements is much larger than these two items.  Rather than opening the door for vague readability complaints, we should either cover readability comprehensively (possibly with a link), or omit talk of readability and break out the two items (variables and comments) individually